HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -1273,7 +1271,119 @@ def self.calc_hrv_erv_effectiveness(vent_mech, vent_mech_cfm, vent_mech_fan_w)
     return vent_mech_sens_eff, vent_mech_lat_eff, vent_mech_apparent_sens_eff
   end
 
-  def self.apply_infiltration_and_ventilation_fans(model, weather, vent_mech, vent_kitchen, vent_bath, has_flue_chimney, air_infils, vented_attic, vented_crawl)
+  def self.get_mech_vent_cfm(vent_mech)
```

Comment:
Maybe we should add a `flow_rate` method to the HPXML VentilationFan class that does this? Then we could always just call that method for every VentilationFan object without worrying about whether it was considered a tested or rated value.